src/cf-core/src/cf-utilities.less

```diff
@@ -229,41 +238,41 @@
 // Margin utilities
 //
 
-.u-mt0  { margin-top:    0 !important; }
-.u-mb0  { margin-bottom: 0 !important; }
-.u-mt5  { margin-top:    5px !important; }
-.u-mb5  { margin-bottom: 5px !important; }
-.u-mt10 { margin-top:    10px !important; }
+.u-mt0 { margin-top: 0 !important; }
```

Comment:
Maybe have stylelint ignore the manual alignments in our margin and width utilities in this file?

Reply:
If the tradeoff is adding code comments to ignore stylelint rules versus having the visual alignment off by a space, I'd go with the visual misalignment. It's not like it's incomprehensible with the values not totally lining up, and having additional code comments seems like clutter that's even worse. Arguably particular lines of code should not depend on the formatting of the lines around them anyway (other than nesting).